x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/formula/validation.ts

```diff
@@ -621,6 +675,24 @@ function runFullASTValidation(
               locations: node.location ? [node.location] : [],
             })
           );
+        } else {
+          const requiredFunctions = nodeOperation.requiredReferences
+            ? nodeOperation.requiredReferences.length
+            : 1;
+          if (functions.length > requiredFunctions) {
+            errors.push(
+              getMessageFromId({
+                messageId: 'tooManyFirstArguments',
```

Comment:
The message still says `a single ...`, but I think it's fine for now - as you mentioned the parsing logic can't handle this either at the moment.